prod/sst/otel.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable */
+/// <reference path="./.sst/platform/config.d.ts" />
+interface ContainerDefinition {
+  name: string;
+  image?: string;
```

Comment:
What happens if `image` is not specified?

Reply:
its sometimes optional because a image can be defined a level above the container definition